x-pack/plugins/security_solution/common/api/search_strategy/cti/event_enrichment.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+import { CtiQueries } from '../model/factory_query_type';
+import { requestBasicOptionsSchema } from '../model/request_basic_options';
+import { timerange } from '../model/timerange';
+
+export const eventEnrichmentRequestOptionsSchema = requestBasicOptionsSchema.extend({
+  eventFields: z.record(z.unknown()),
```

Comment:
is the `z.unknown()` necessary?

Reply:
changed to string as it probably makes more sense here